docs/_documentations/remotedeploy-combo.md

```diff
@@ -10,59 +10,64 @@ parent: installoncloud
 order: 2
 ---
 
-## Installing a Codewind remote deployment
+# Installing a Codewind remote deployment all-in-one
 
-Codewind includes a CLI to simplify the installation process. You will find `cwctl` in your HOME directory under the path ~/.codewind/{version}
+Codewind includes a CLI to simplify the installation process. You can find the `cwctl` CLI in your HOME directory under the path `~/.codewind/{version}`.
 
-The command for installing an "all-in" deployment with a new Keycloak and new Codewind service is :
+The command for installing an all-in-one deployment with a new Keycloak and a new Codewind service is:
 
 `cwctl --insecure install remote` 
 
-and requires various flags to specify where and what to install.
+This command requires various flags to specify where and what to install.
 
-### Deploying Codewind with CWCTL
+## Deploying Codewind with CWCTL
 
-1.  Open a new terminal window on your local workstation
-2.  Navigate to your home directory and to the Codewind cli:
+1.  Open a new terminal window on your local workstation.
+2.  Navigate to your home directory, and then to the Codewind CLI:
```

Comment:
Comma not needed after "directory."
The phrase "then to the Codewind CLI" is not an independent clause that can stand alone. Thus, the conjunction "and" can alone suffice to separate it from the rest of the sentence.